---
layout: post
title: research
cover: Yeast.jpg
date:   2015-04-30 12:00:00
categories: posts
---

## Evolution of the Nervous System

When did the nervous system originate and what were the key changes associated with this important evolutionary
event? My PhD work explored this question by investigating the evolutionary history of voltage-gated ion channels,
the main drivers of electrical excitability in nervous systems. I did this work in the labs of 
[Harold Zakon](http://www.biosci.utexas.edu/neuroscience/HaroldZakon/) and [David Hillis](http://www.zo.utexas.edu/faculty/antisense/)
at University of Texas at Austin. 

We recently wrote two reviews, together with [Hans Hofmann](https://cichlid.biosci.utexas.edu/), about
the state of this growing sub-field. [One](http://www.cell.com/trends/ecology-evolution/fulltext/S0169-5347(15)00302-X), 
in Trends in Ecology and Evolution, explores the question of homology and homoplasy in complex molecular systems. The
[other](http://www.annualreviews.org/doi/abs/10.1146/annurev-ecolsys-110316-023048), in Annual Reviews of Ecology, Evolution
and Systematics, discusses at greater length the molecular evolution of nervous systems (or, as we prefer, "neural systems")
and why the field needs better sampling of functional data across species.

As a postdoc with [Edward Marcotte](http://www.marcottelab.org/index.php/Main_Page) and [Rick Aldrich](https://clm.utexas.edu/AldrichLab/),
I've been developing proteomics and computational methods to do just that. I use a combination of mass-spectrometry, machine learning, and 
phylogenetic modeling to build protein-interaction maps across species, with special emphasis on my beloved ion channels and other neural
proteins.

You can also check out a blog I wrote for [BEACON](http://beacon-center.org/)
a few years back: [Electrical Life](http://beacon-center.org/blog/2013/10/14/beacon-researchers-at-work-electrical-life/)

## The Comparative Approach to Human Disease

Organismal diversity has two main, non-independent causes: shared ancestry (history), and the evolutionary process (selection).
How can we leverage these two causes to better understand the genetics of human disease? I recently wrote a [review of this 
subject](http://www.sciencedirect.com/science/article/pii/S0959437X15000878/) that explores the variety of ways that
researchers leverage organismal diversity, model systems, and evolutionary reasoning to discover new genes associated with
human diseases. 

## Evolutionary Genomics

One of my abiding interests is in reconstructing the history of gene families and in correlating this history with major changes
in morphology or physiology. For instance, [this paper](http://www.pnas.org/content/112/8/E846) explores the history of ion channel
gene families, which radiated independently in a number of early animal lineages. Did these independent radiations cause convergent
neural phenotypes in cnidarians, ctenophores, and bilaterians?

One common way that researchers track the history of gene families is to "flatten" the phylogenetic tree into groups corresponding to 
sets of homologs (as in "phylostratigraphy"](https://en.wikipedia.org/wiki/Genomic_phylostratigraphy)) or orthologs. We [recently showed]
(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4943184/) that common orthology inference algorithms, which are used upstream of many other 
phylogenetic applications as well, disagree strongly with one another when used to infer gene ages. We found evidence of widespread systematic
error that correlated with algorithm type, and suggested that any application using these algorithms ought to employ some sort of consensus
strategy, and ideally should propagate uncertainty into the downstream analysis. This project was made possible by the 
[Quest for Orthologs](https://questfororthologs.org/) consortium, a group of orthology researchers with a committment to open data,
 community standards, and benchmarking -- a model scientific community.